src/Files.App/UserControls/Widgets/FileTagsWidget.xaml

```diff
@@ -98,6 +98,7 @@
 						ItemClick="FileTagItem_ItemClick"
 						ItemsSource="{x:Bind Tags}"
 						RightTapped="AdaptiveGridView_RightTapped"
+						ScrollViewer.VerticalScrollBarVisibility="Hidden"
```

Comment:
I have found that displaying nested scrollbars in ScrollViewer causes a crash. So, as a workaround, I hide the scrollbars in the tags widget.
Even without scrollbars, it is possible to scroll with the mouse wheel or touchpad gestures.

Reply:
Is there anywhere else we should do this?

Reply:
NOTICE.md in the About page also has a nested scroll bar. But it doesn't seem to crash.